packages/runtime-common/loader.ts

```diff
@@ -115,7 +117,7 @@ export type RequestHandler = (req: Request) => Promise<Response | null>;
 export class Loader {
   private log = logger('loader');
   private modules = new Map<string, Module>();
-  private urlHandlers: RequestHandler[] = [];
+  private urlHandlers: RequestHandler[] = [maybeHandleScopedCSSRequest];
```

Comment:
Does it make sense for this handler to always be present?

Reply:
i think that seems fine